### PR TITLE
[application] Thibaut

### PR DESCRIPTION
**Pull request by Tbaut**

### CODE REVIEW

File: join_request/tbaut [hunk 1]
```diff
@@ -0,0 +1,34 @@
+collective: Potoc
+member:
+  name: Thibaut Sardan
+  address: 1ThiBx5DDxFhoD9GY6tz5Fp4Y7Xn1xfLmDddcoFQghDvvjg
+  github: tbaut
```
Comment (wirednkod) [inline join_request/tbaut]: please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 1ThiBx5DDxFhoD9GY6tz5Fp4Y7Xn1xfLmDddcoFQghDvvjg
  matrix:
  github: tbaut
```

Comment (Tbaut) [reply join_request/tbaut]:
```suggestion
  address: 1ThiBx5DDxFhoD9GY6tz5Fp4Y7Xn1xfLmDddcoFQghDvvjg
  matrix: @tbaut:matrix.org
  github: tbaut
```

File: join_request/tbaut [hunk 2]
```diff
@@ -0,0 +1,35 @@
+collective: Potoc
+member:
+  name: Thibaut Sardan
+  address: 1ThiBx5DDxFhoD9GY6tz5Fp4Y7Xn1xfLmDddcoFQghDvvjg
+  matrix: @tbaut:matrix.org
```
Comment (Tbaut) [inline join_request/tbaut]:
```suggestion
  matrix: "@tbaut:matrix.org"
```

**Comment by Tbaut** [comment]
closing as the projects I'm willing to work on aren't part of the definitive list :(

**Pull request closed without merging**